atom/browser/resources/mac/Info.plist

```diff
@@ -32,5 +32,9 @@
   <true/>
   <key>NSHighResolutionCapable</key>
   <true/>
+  <key>NSMicrophoneUsageDescription</key>
+	<string></string>
```

Comment:
- indentation is weird
- put a dummy description here? I'm not quite sure what the dialog looks like so I'm not sure what would make sense